plugins/obs-transitions/transition-luma-wipe.c

```diff
@@ -5,10 +5,12 @@
 #define S_LUMA_IMG              "luma_image"
 #define S_LUMA_INV              "luma_invert"
 #define S_LUMA_SOFT             "luma_softness"
+#define S_LUMA_COLOR		"luma_color"
```

Comment:
Style:  Use spaces for alignment instead of tabs.

```suggestion
#define S_LUMA_COLOR            "luma_color"
```